src/app/modules/main/profile_section/contacts/view.nim

```diff
@@ -204,4 +239,79 @@ QtObject:
     self.delegate.requestContactInfo(publicKey)
 
   proc onContactInfoRequestFinished*(self: View, publicKey: string, ok: bool) {.slot.} =
-    self.contactInfoRequestFinished(publicKey, ok)
\ No newline at end of file
+    self.contactInfoRequestFinished(publicKey, ok)
+
+  # Showcase models for a contact
+  proc getShowcaseContactCommunitiesModel(self: View): QVariant {.slot.} =
+    return self.showcaseContactCommunitiesModelVariant
+  QtProperty[QVariant] showcaseContactCommunitiesModel:
+    read = getShowcaseContactCommunitiesModel
+
+  proc getShowcaseContactAccountsModel(self: View): QVariant {.slot.} =
+    return self.showcaseContactAccountsModelVariant
+  QtProperty[QVariant] showcaseContactAccountsModel:
+    read = getShowcaseContactAccountsModel
+
+  proc getShowcaseContactCollectiblesModel(self: View): QVariant {.slot.} =
+    return self.showcaseContactCollectiblesModelVariant
+  QtProperty[QVariant] showcaseContactCollectiblesModel:
+    read = getShowcaseContactCollectiblesModel
+
+  proc getShowcaseContactAssetsModel(self: View): QVariant {.slot.} =
+    return self.showcaseContactAssetsModelVariant
+  QtProperty[QVariant] showcaseContactAssetsModel:
+    read = getShowcaseContactAssetsModel
+
+  proc getShowcaseContactSocialLinksModel(self: View): QVariant {.slot.} =
+    return self.showcaseContactSocialLinksModelVariant
+  QtProperty[QVariant] showcaseContactSocialLinksModel:
+    read = getShowcaseContactSocialLinksModel
+
+  # Support models for showcase for a contact
+  proc showcaseCollectiblesModelChanged*(self: View) {.signal.}
+  proc getShowcaseCollectiblesModel(self: View): QVariant {.slot.} =
+    return self.delegate.getShowcaseCollectiblesModel()
+  QtProperty[QVariant] showcaseCollectiblesModel:
+    read = getShowcaseCollectiblesModel
+    notify = showcaseCollectiblesModelChanged
+
+  proc showcaseForAContactLoadingChanged*(self: View) {.signal.}
+  proc emitShowcaseForAContactLoadingChangedSignal*(self: View) =
+    self.showcaseForAContactLoadingChanged()
+  proc isShowcaseForAContactLoading*(self: View): bool {.signal.} =
+    return self.delegate.isShowcaseForAContactLoading()
+  QtProperty[QVariant] showcaseForAContactLoading:
```

Comment:
@alexjba @benjthayer @noeliaSD After I fixed the issue with the hang when querying the community, there is a problem: at the moment of opening the profile dialog we don't know how many items we will have and validation itself can take significant time. I propose to add this flag to show loading placeholder for all profile showcase entities

Reply:
https://github.com/status-im/status-desktop/assets/2522130/39323895-1cfd-4c3e-bf77-69424e182a96

Reply:
Not sure if I'm missing some subtile detail here, but my proposal is to go the other way: Expose all showcase community data and then filter out those that failed validation (ideally in status-go, but can be done in the UI as well if we have the necessary data). IMO it's more flexible and there's no need for a `loading` flag.

Initially all communities will be shown as loading in the UI because there's no community data. Then when each community is loaded and validated we'll have the proper data to display.
Whenever a community membership validation fails we'll just see the specific card being removed (previously had the loading state). Ideally the validation happens before the community data is propagated in nim.

Having the `loading` flag is also a good option IMO. So if you find it necessary I'm all for it! I'd just rename it to something more specific so that it doesn't conflict with other models roleNames.

Reply:
If we are ok if some entries would disappear, i'll break profile request on two stages. Yea, let's leave `isShowcaseForAContactLoading` it may be useful in the future. For the specific entry, loading  = not presentet in the support model

Reply:
> Not sure if I'm missing some subtile detail here, but my proposal is to go the other way: Expose all showcase community data and then filter out those that failed validation (ideally in status-go, but can be done in the UI as well if we have the necessary data). IMO it's more flexible and there's no need for a `loading` flag.
> 
> Initially all communities will be shown as loading in the UI because there's no community data. Then when each community is loaded and validated we'll have the proper data to display. Whenever a community membership validation fails we'll just see the specific card being removed (previously had the loading state). Ideally the validation happens before the community data is propagated in nim.
> 
> Having the `loading` flag is also a good option IMO. So if you find it necessary I'm all for it! I'd just rename it to something more specific so that it doesn't conflict with other models roleNames.

I'm not sure if it's bc of this l`oading all communities and then filter behaviour` but I've seen the following situation (and I cannot reproduce again):
- First time I tried to look for a CONTACT communities profile (this conntact had 2 communities exposed for everyone and 1 for only contacts), I've seen in a short period the loading state for the 3 communities, then I saw the complete communities loaded (the 3 ones with all data) and then 2 disapeared. 
- Now the state of the profile showcase for this specific contact is this:

https://github.com/status-im/status-desktop/assets/97019400/7bc68b6d-edd2-41be-b65a-e30f7d4b3530

So:
1. Weird loading + displaying all communties + filter some communtites that where already "revealed"
2. It seems something is wrong with the visibility level

Reply:
Here in the following video I have been able to grasp a little of what I explain to you:

https://github.com/status-im/status-desktop/assets/97019400/72ecd5c8-799d-40f5-90db-2360976682d4

1. It's not been as evitent than before, the filtered community has been hidden quicker this time, but you see the data is revealed. In the other try, the communities were there for some seconds so I could read them more or less correctly. 
2. However, here I have the same issue with the visibilities.. I only see one community when the contact is exposing 3.

Reply:
Yes, looks like it! But if we have the loading flag we'll take that into account as well. It will show the loading state of the community and then it will dissapear.

Reply:
Here another example (more evident the data revealed but then hidden):

https://github.com/status-im/status-desktop/assets/97019400/15cf7f9c-4edb-4387-9119-b235385b2b6b

> Yes, looks like it! But if we have the loading flag we'll take that into account as well. It will show the loading state of the community and then it will dissapear.

Yes, I agree on the approach but I think we need to ensure the data is not displayed if it must be filtered out. However I'm not sure that my videos are a good example bc as you see, something with the visibilities is wrong so.. this can cause also this data beeing displayed for some seconds.. bc indeed.. these communities should NOT be filtered out.